packages/ui/src/components/va-input/VaInput.stories.ts

```diff
@@ -1,12 +1,48 @@
 import { defineComponent } from 'vue'
-import VaInput from './VaInput.demo.vue'
+import VaInputDemo from './VaInput.demo.vue'
+import { VaInput } from './'
+import { expect } from '@storybook/jest'
+import type { StoryFn } from '@storybook/vue3'
 
 export default {
   title: 'VaInput',
   component: VaInput,
 }
 
 export const Default = defineComponent({
+  components: { VaInputDemo },
+  template: '<VaInputDemo />',
+})
+
+export const Disabled: StoryFn = () => ({
+  components: { VaInput },
+  template: '<VaInput disabled/>',
+})
+Disabled.play = async ({ canvasElement, step }) => {
```

Comment:
I don't think we need interaction demo here. This prop is completely visual. (or should be visual)